app/bundles/CoreBundle/Tests/Unit/Command/ModeratedCommandTest.php

```diff
@@ -234,7 +234,7 @@ public function testRedisLock(): void
     {
         $this->coreParametersHelper->expects($this->once())
             ->method('get')
-            ->willReturn(['dsn' => null]);
+            ->willReturn(['dsn' => '']);
```

Comment:
@kuzmany would be great if you can add a sample DSN instead of an empty string, so other developers can get a sense of how this string should look like

Reply:
If I add it it return error because cannot connect to fake server  (I spent a lot of time on it, cannot do more...) https://github.com/mautic/mautic/actions/runs/4914285566/jobs/8775410359?pr=12303#step:10:81

Do you have ideas to fix it? I make unit tests just to be sure If redis lock goes to the proper case of code.

Reply:
I revert it back. Unit tests do what should do. I check If  `--lock=redis` go to proper way